src/ploneintranet/search/README.rst

```diff
@@ -0,0 +1,24 @@
+Plone Intranet Search
+=====================
+
+This package aims to provide a common API
+for indexing and querying from Plone
+to a 3rd party search engines.
```

Comment:
Grammatical error here. I believe this should read either:
"to a 3rd party search engine"
or
"to 3rd party search engines"